### PR TITLE
add validator exception wrapper method/decorator 

### PR DESCRIPTION
**Pull request by freshkevin**
Makes validators raise exceptions if they return False.

### CODE REVIEW

File: openhtf/util/validators.py [hunk 1]
```diff
@@ -62,6 +62,14 @@ def MyPhase(test):
 
 _VALIDATORS = {}
 
+class ValidatorException()
```
Comment (michelleat) [inline openhtf/util/validators.py]: is this valid syntax?  when i run it on its own it doesn't execute.  Also, should you inherit from the Exception base class?

File: openhtf/util/validators.py [hunk 2]
```diff
@@ -62,6 +62,14 @@ def MyPhase(test):
...
+
+def wrap_call_exception(validator):
+  def wrapper(*args, **kwargs):
+    result = validator(args, kwargs)
+    if not result:
+      raise ValidatorException("%s conditions not met\nargs: %s kwargs: %s" % (validator.__name__, args, kwargs))
+    return result
```
Comment (michelleat) [inline openhtf/util/validators.py:73]: should this be returning the wrapped function?

File: openhtf/util/validators.py (hunk 1 above)
Comment (freshkevin) [reply openhtf/util/validators.py]: Yep, should be inheriting from Exception. I clearly remember thinking that I needed to remember to do that and obviously I did not remember hard enough.

File: openhtf/util/validators.py (hunk 2 above)
Comment (freshkevin) [reply openhtf/util/validators.py:73]: Yup...

**Comment by freshkevin** [comment]
Updated to fix referenced errors.

File: openhtf/util/validators.py (hunk 2 above)
Comment (michelleat) [reply openhtf/util/validators.py:73]: cooool.  and one more thing  :)   i think you also need asterisks for validator(*args, **kwargs).

**Pull request merged**